releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,11 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.2.7 - July 14, 2020
+
+* Fixed the bug where "featureManagement." needed to be added before some feature flag names when coming from App Configuration.
```

Comment:
I cannot understand what this is talking about :(

Reply:
Fixed the bug feature flags from App Configuration do not work.

or

Fixed the bug conditional feature flags from App Configuration do not work.

Reply:
Fixed the bug that feature flags that do not use feature filters will appear always off regardless of their actual states.